src/elm/Pages/Secrets/Form.elm

```diff
@@ -181,6 +196,7 @@ viewEventsSelect secretUpdate =
         , div
             [ class "form-controls"
             , class "-stack"
+            , Util.testAttribute "event-checkbox"
```

Comment:
do we still need this? doesn't look used in this PR.

Reply:
Nope, removed!